managed/devops/opscli/ybops/cloud/aws/method.py

```diff
@@ -207,7 +207,7 @@ def callback(self, args):
         if not host_info:
             logging.error("Host {} does not exist.".format(args.search_pattern))
             return
-        self.cloud.start_instance(host_info)
+        self.cloud.start_instance(host_info, int(args.custom_ssh_port) or 22)
```

Comment:
` self.parser.add_argument("--custom_ssh_port",
                                 required=False,
                                 help="The ssh port to connect to.")`
  this is not required arg so is this change looks fine? I can see 54422 going on if we create a provider without sshport.

Reply:
` commandArgs.addAll(getAccessKeySpecificCommand(taskParam, type));`  adding `custom_ssh_port`  in this method.
so guess we will always have value for it.
Is it safe to remove 22?